phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,7 +39,11 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(applicationName, themeConfiguration.themeManager(), phoenicisGlobalConfiguration.objectMapper());
+        return new ViewLibrary(
```

Comment:
In addition how do you run the plugin?
If I run it, the plugin is working fine (the given parameters are split to two lines).
I did run the plugin via `mvn test`